sentry-samples/sentry-samples-console/src/main/java/io/sentry/samples/console/Main.java

```diff
@@ -0,0 +1,114 @@
+package io.sentry.samples.console;
+
+import io.sentry.core.Breadcrumb;
+import io.sentry.core.EventProcessor;
+import io.sentry.core.Sentry;
+import io.sentry.core.SentryEvent;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.protocol.Message;
+import io.sentry.core.protocol.User;
+import java.util.Collections;
+
+public class Main {
+
+  public static void main(String[] args) {
+    Sentry.init(
+        options -> {
+          // NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in
+          // your Sentry project/dashboard
+          options.setDsn("https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954");
+
+          // Modifications to event before it goes out. Could replace the event altogether
+          options.setBeforeSend(
+              (event, hint) -> {
+                // Drop an event altogether:
+                if (event.getTag("SomeTag") != null) {
+                  return null;
+                }
+                return event;
+              });
+
+          // Allows inspecting and modifying, returning a new or simply rejecting (returning null)
+          options.setBeforeBreadcrumb(
+              (breadcrumb, hint) -> {
+                // Don't add breadcrumbs with message containing:
+                if (breadcrumb.getMessage() != null
+                    && breadcrumb.getMessage().contains("bad breadcrumb")) {
+                  return null;
+                }
+                return breadcrumb;
+              });
+
+          // Configure the background worker which sends events to sentry:
+          // Wait up to 5 seconds before shutdown while there are events to send.
+          options.setShutdownTimeout(5000);
+
+          // Enable SDK logging with Debug level
+          options.setDebug(true);
+          // To change the verbosity, use:
+          // By default it's DEBUG.
+          options.setDiagnosticLevel(SentryLevel.ERROR); //  A good option to have SDK debug log in prod is to use only level ERROR here.
+
+          // Using a proxy:
+          options.setProxy(null); // new Proxy(Type.HTTP, new InetSocketAddress(8090)));
```

Comment:
I've never tested this class, to be honest, it's pretty much a copy-paste from sentry-java, it's fine to keep here but recently @thinkocapo could not use it for some reason and I still need to figure out if it's a bug or a proxy setup.

Reply:
Removed in #502 as I believe we should have only working and tested code in sample. We can add it once we gain more confidence.